app/javascript/react/screens/App/common/ListViewToolbar/ListViewToolbar.js

```diff
@@ -125,7 +125,7 @@ class ListViewToolbar extends Component {
     return listFilter(activeFilters, listItems);
   };
 
-  filterSortPaginateListItems = (filteredItems = this.filterListItems()) => {
+  filterSortPaginateListItems = (filteredItems = [...this.filterListItems()]) => {
```

Comment:
I wonder if we should instead do the copying inside `sortFilter` itself, immediately before the `[].sort()` calls, so we prevent the mutation from happening in other places where that function is used? I could go either way on this, since the impact of that would be harder to test.

Reply:
I wasn't exactly sure where the best place to do this was either.  If you look at `listFilter`, if there are active filters, the method returns a new array, so no danger of state mutation.  However, if there aren't any filters, it just returns the original array.

I didn't want to bury this inside the helper, but if you think that's the better approach, we can go that route

Reply:
I guess we shouldn't bury it in the helper, just in case that has some unintended (removal of) side effects. Considering we want to eventually replace all uses of that helper with `ListViewToolbar` anyway, this is good.

Reply:
Wait a minute, your'e totally right.  It's the helper's responsibility to make the copy b/c it makes use of the `sort` method.  I'll change this in the morning